packages/ogmios/src/Provider/TxSubmitProvider/OgmiosTxSubmitProvider.ts

```diff
@@ -63,10 +66,28 @@ export class OgmiosTxSubmitProvider extends RunnableModule implements TxSubmitPr
     }
   }
 
-  async submitTx({ signedTransaction }: SubmitTxArgs): Promise<void> {
+  async submitTx({ signedTransaction, context }: SubmitTxArgs, handleProvider?: HandleProvider): Promise<void> {
     if (this.state !== 'running') {
       throw new CardanoNodeErrors.NotInitializedError('submitTx', this.name);
     }
+
+    if (context && context.handles && context.handles.length > 0) {
```

Comment:
```suggestion
    if (context?.handles?.length > 0) {
```

Reply:
If I do this, the ternary can return an `undefined`, but handles don't take `undefined`.. it takes a `handle: HandleResolution[]` or an empty `[]`. So the first check would return a `null` if context and context handles don't exist. Which is more suited to this situation? Otherwise, I need to modify handles to take undefined, but in previous work it was specified not to do it this way to better control the data we have in different scenarios..

Reply:
Superseded by: https://github.com/input-output-hk/cardano-js-sdk/pull/761#discussion_r1230742567 and https://github.com/input-output-hk/cardano-js-sdk/pull/761#discussion_r1230742819